### DISCUSSION TITLE
actualizar datos de XTOTV canal de Bolivia

### DISCUSSION THREAD

**Discussion by LEODJ69** [iptv-org/community#1104] [Q&A]
Por favor podria actualizar datos de XTOTV canal Boliviano?
WEB: https://www.sccbolivia.com/
URL TRASMICION : http://190.104.15.135/0.ts
Logo: https://www.sccbolivia.com/wp-content/uploads/2020/09/xtotv-tx-inicio.png

muchas gracias!

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/5165b3b06d6391e19ae8134885593845d96e3918/streams/bo.m3u#L138-L142
